server/model/newExamResultsSubscriber.go

```diff
@@ -0,0 +1,14 @@
+package model
+
+import (
+	"database/sql"
+	"time"
+)
+
+type NewExamResultsSubscriber struct {
+	ID             int32          `gorm:"primary_key;AUTO_INCREMENT;" json:"id"`
+	CallbackUrl    string         `json:"callbackUrl"`
```

Comment:
In your next PR this changes the primary key to `CallbackUrl`. Do you think we should do this off the start?